crates/turbopack-core/src/source_map/mod.rs

```diff
@@ -43,7 +43,7 @@ pub trait GenerateSourceMap {
 /// The distinction between the source map spec's [sourcemap::Index] and our
 /// [SourceMap::Sectioned] is whether the sections are represented with Vcs
 /// pointers.
-#[turbo_tasks::value(shared)]
+#[turbo_tasks::value(shared, cell = "new")]
```

Comment:
`cell = "new"` changes the behavior of `.cell()` to no longer perform an `==` check and just always override the cell.

Reply:
How do we determine which cells should always be overridden?

Reply:
When you don't want cells to ever match using equality? Which I imagine should be never unless you've got another bug (e.g. inconsistent cell creation order?).

I agree this seems weird and like we're masking bigger problems.

Reply:
> When you don't want cells to ever match using equality?

* some types have every expensive equal checks and are very unlikely to be equal (e. g. the parsed AST or here the SourceMap)
* some types don't have a equality defined
* some types define equality in a way that it never matches

But in general you want to check equality for early stop bubbling invaliations...